src/shared/lib/onMobxPromise.ts

```diff
@@ -1,15 +1,32 @@
 import {MobxPromise} from "mobxpromise";
 import {autorun, IReactionDisposer} from "mobx";
 
-export default function onMobxPromise<T>(promise:MobxPromise<T>,
-                                         onComplete:(result:T)=>void,
+export function onMobxPromise<T>(promise:MobxPromise<T>,
+                          onComplete:(result:T)=>void,
```

Comment:
onError?

Reply:
I havent had a need for that yet so I didn't implement it